src/Sdk/Validation/Rules/GreaterThan.php

```diff
@@ -0,0 +1,22 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\SdkBlueprint\Sdk\Validation\Rules;
+
+use LoyaltyCorp\SdkBlueprint\Sdk\Validation\Rule;
+
+class GreaterThan extends Rule
+{
+    /**
+     * 'greaterThan' rule
+     *
+     * @return void
+     */
+    protected function process(): void
+    {
+        // If the value is less than or equal to required, validation fails
+        if ($this->hasValue() && (int)$this->getValue() <= (int)$this->parameters) {
```

Comment:
Is there a reason we couldn't support decimals/floats with these numeric rules? It could be done easily using BCMath functions

Reply:
This comes from the v1, as mentioned in the doc https://github.com/loyaltycorp/sdkblueprint it looks like this is for integer. @{d433845b-4824-46f7-a4e5-e198e09c023c,Nathan Page}, what do you think?

Reply:
Yes the reason is that the v1 has been developed in a really tie timeframe :) If someone has an easy and efficient way to improve it let's go for it ;)